Common/src/Storage/TaskOptions.cs

```diff
@@ -33,39 +33,14 @@ public record TaskOptions(IDictionary<string, string> Options,
                           string                      ApplicationService,
                           string                      EngineType)
 {
-  public static implicit operator Api.gRPC.V1.TaskOptions(TaskOptions taskOption)
```

Comment:
cf issue mentionned above

Reply:
Here is the comment:
> and break nullable. It would generate a lot of warnings.
> 
> And there is a great issue I encountered:
> 
>     * CreateTask takes an option and an `IAsyncEnumerable<TaskRequest>`
> 
>     * the conversion from gprc options to internal options does not work because the grpc options is null. It comes from the fact that the implicit conversion does not work well with null (we cannot easily detect that the object is null before the implicit conversion and avoid making this conversion that does not work. Doing so takes a lot of lines of code that are not really self-explanatory)
> 
>     * so implicit conversion does not work but we still enters task creation through the IAsyncEnumerable
> 
>     * request are not consummed and application wait
> 
>     * task creation handle indefinitely.

Fair enough, I just wanted to know if there was any technical reason or just style preference.